### PR TITLE
Document model_path

### PR DESCRIPTION
**Pull request by thomaschaaf**
model_path was added with the PR https://github.com/propelorm/Propel2/pull/1125

### CODE REVIEW

File: documentation/reference/configuration-file.markdown [hunk 1]
```diff
@@ -218,6 +218,17 @@ The `slaves` properties groups lists slave `connection` elements which provide s
                   - ......
 ```
 
+#### Model path
+
+`model_paths` specifies the folders where the models are located. This can help improve some operations. The default folders are `srv` and `vendor`.
+
+```yaml
+              model_paths:
+                  - srv
```
Comment (marcj) [inline documentation/reference/configuration-file.markdown]: src ?

**Comment by thomaschaaf** [comment]
Sorry fixed

File: documentation/reference/configuration-file.markdown [hunk 2]
```diff
@@ -218,6 +218,17 @@ The `slaves` properties groups lists slave `connection` elements which provide s
                   - ......
 ```
 
+#### Model path
+
+`model_paths` specifies the folders where the models are located. This can help improve some operations. The default folders are `src` and `vendor`.
```
Comment (marcj) [inline documentation/reference/configuration-file.markdown]: perhaps we should a note that this is mainly used for fixtures, isn't it?

**Comment by robin850** [comment]
Thanks @thomaschaaf !

@marcj : Sorry, I forgot to squash the commits merging this. 😓

**Pull request merged**